services/api-server/src/simcore_service_api_server/services/storage.py

```diff
@@ -95,6 +95,13 @@ async def get_download_link(
         link: AnyUrl = presigned_link.link
         return link
 
+    async def delete_file(self, user_id: int, quoted_storage_file_id: str) -> None:
+        response = await self.client.delete(
+            f"{self.client.base_url}locations/{self.SIMCORE_S3_ID}/files/{quoted_storage_file_id}",
+            params={"user_id": user_id},
+        )
+        response.raise_for_status()
```

Comment:
We need to review carefully error handling. We should have a careful mapping between this client errors and the api-server errors. This is not for this PR but it is an opportunity to show you one big defect I am trying to systematically fix. I will share with you some of my thoughts separately.